webdriver/tests/user_prompts/get_alert_text.py

```diff
@@ -0,0 +1,50 @@
+from tests.support.asserts import assert_error, assert_success
+
+
+def get_dialog_text(session):
+    return session.transport.send("GET", "session/{session_id}/alert/text"
+                                  .format(session_id=session.session_id))
+
+
+# 18.3 Get Alert Text
+
+def test_no_browsing_context(session, create_window):
+    # 18.3 step 1
+    session.window_handle = create_window()
+    session.close()
+
+    response = get_dialog_text(session)
+    assert_error(response, "no such window")
+
+
+def test_no_user_prompt(session):
+    # 18.3 step 2
+    response = get_dialog_text(session)
+    assert_error(response, "no such alert")
+
+
+def test_get_alert_text(session):
+    # 18.3 step 3
+    session.execute_script("window.alert(\"Hello\");")
+    response = get_dialog_text(session)
+    alert_text = response.body["value"]
+    assert alert_text == "Hello"
+    assert_success(response)
```

Comment:
As above, this needs to be done first, preferably with `assert response.status == 200` in this case.

Reply:
fixed